src/main/osd/osd_elements.c

```diff
@@ -997,28 +1000,28 @@ static void osdElementFlymode(osdElementParms_t *element)
     //  5. ACRO
 
     if (FLIGHT_MODE(FAILSAFE_MODE)) {
-        strcpy(element->buff, "!FS!");
+        strcpy(element->buff, STR_ODSE_FLYMODE_FAILSAFE);
```

Comment:
Is ODSE intentional?

Reply:
OSDE are short for OSD_Element.c, OSD for OSD.c and OSDW for OSD_Warning.c

ODSE are a good catch (why I didn't saw that ??...), I change to OSDE*

Reply:
Fixed :-)